website_sale_charge_payment_fee/models/sale_order.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 AITIC S.A.S
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    amount_payment_fee = fields.Monetary(
+        compute="_compute_amount_payment_fee",
+        digits=0,
+        string="Payment Fee Amount",
+        store=True,
+        track_visibility="always",
+    )
```

Comment:
Would be nice to mention that the field (and logic) are made consistent with amount_delivery from website_sale_delivery.